VerloopSDK/VLWebViewManager.swift

```diff
@@ -157,3 +237,243 @@ class VLWebViewManager: NSObject, WKScriptMessageHandler, WKUIDelegate, WKNaviga
         }
     }
 }
+
+
+extension VLWebViewManager {
+    
+    private func processRoomReadyConfigurations() {
+        for config in roomReadyConfigurations {
+            switch config {
+                case .close :
+                    webView.evaluateJavaScript(String.getCloseEvaluateJS()) { _, error in
+                        print("getCloseEvaluateJS error \(error?.localizedDescription ?? "NA")")
+                    }
+                case .closeWidget:
+                    webView.evaluateJavaScript(String.getWidgetClosedEvaluationJS()) {[weak self] _, error in
+                        print("closeWidget error \(error?.localizedDescription ?? "NA")")
+                        if error == nil {
+//                            self?._eventDelegate?.didEventOccurOnLiveChat(.onChatMinimized)
+                        }
+                    }
+                default: break
+            }
+        }
+        roomReadyConfigurations = []
+    }
+    
+    private func processNetworkChangeConfigurations() {
```

Comment:
check this

Reply:
A subset of callbacks are triggered on network change. Hence we are using this method. Commenting here to keep track.